src/test/regress/expected/multi_create_table.out

```diff
@@ -122,15 +122,24 @@ CREATE TABLE primary_key_on_non_part_col
 );
 SELECT master_create_distributed_table('primary_key_on_non_part_col', 'partition_col', 'hash');
 ERROR:  cannot distribute relation: "primary_key_on_non_part_col"
-DETAIL:  Distributed relations cannot have UNIQUE constraints or PRIMARY KEYs that do not include the partition column.
+DETAIL:  Distributed relations cannot have UNIQUE constraints, EXCLUDE constraints, or PRIMARY KEYs that do not include the partition column.
 CREATE TABLE unique_const_on_non_part_col
 (
 	partition_col integer,
 	other_col integer UNIQUE
 );
-SELECT master_create_distributed_table('primary_key_on_non_part_col', 'partition_col', 'hash');
-ERROR:  cannot distribute relation: "primary_key_on_non_part_col"
-DETAIL:  Distributed relations cannot have UNIQUE constraints or PRIMARY KEYs that do not include the partition column.
+SELECT master_create_distributed_table('unique_const_on_non_part_col', 'partition_col', 'hash');
+ERROR:  cannot distribute relation: "unique_const_on_non_part_col"
+DETAIL:  Distributed relations cannot have UNIQUE constraints, EXCLUDE constraints, or PRIMARY KEYs that do not include the partition column.
```

Comment:
If we're to implement my suggestion above, this error message becomes obsolete for `EXCLUDE` constraints. We should probably have two different error messages, where in `EXCLUDE` constraints' error message we mention that partition column should be specified with an equality operator.